lib/action_view/component/base.rb

```diff
@@ -50,6 +50,10 @@ def render_in(view_context, *args, &block)
 
         return "" unless render?
 
+        render_template(&block)
+      end
+
+      def render_template(&block)
```

Comment:
You might want to add some documentation here about the expected use case of a sub-class overriding this method to do caching.

Reply:
Thanks! This is much better!